Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/WSDLParser.cs

```diff
@@ -97,19 +97,31 @@ public override ObservableList<ApplicationAPIModel> ParseDocument(string URL, Ob
                 {
                     foreach (Binding b in SDExtended.mServiceDescription.Bindings)
                     {
-                        bindColl.Add(b);
+                        if (bindColl[b.Name] == null)
```

Comment:
why changes to WSDLParser were needed? you found bug?

Reply:
Yes, and it's fixed here. I opened it last week as below;
Bug 8003:Relearn/Refresh WSDL APIs fails and generates error as, "Error while parsing"
As on re-learn it was trying to re-add/bind the already existing columns from the link, which was creating issue here.